project/profiles/capacity4more/modules/c4m/og/c4m_og/c4m_og.module

```diff
@@ -685,3 +691,75 @@ function c4m_og_get_user_group_nodes_by_bundle(
   }
   return $groups;
 }
+
+/**
+ * Implements hook_node_update().
+ */
+function c4m_og_node_update($node) {
+  if ($node->type != 'group') {
+    return;
+  }
+
+  c4m_og_open_group_access_activate_pending_members($node);
+}
+
+/**
+ * Activate pending members when access level of a group is changed.
+ *
+ * When changing the access level from 'Moderated' to 'Open', all pending
+ * members should become active group members.
+ *
+ * @param object $node
+ *   The node object from the "hook_node_update".
+ */
+function c4m_og_open_group_access_activate_pending_members($node) {
+  require_once DRUPAL_ROOT . '/sites/all/modules/contrib/og/includes/actions/set_state.action.inc';
```

Comment:
even if we needed this, it would be wrong to hardcode the path. You probably look for `module_load_include()`. But anyway, I don't think we need the action.